tools/sklearn/test-data/pca_input.dat

```diff
@@ -0,0 +1,300 @@
+9.579515262220434924e-01 3.726315038026500881e-02 1.192922318396479886e+00 2.208390853256675612e+00 1.545465138214503487e+00 6.587709093437858598e+00 2.676346016926679283e+00 2.002623182927643519e+00 1.960111203222609788e+00 1.244000891424809074e-01
```

Comment:
A small thing - can you also update this file to be tab separated? I think the sklearn input macros require it.